app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate params[:session][:password]
```

Comment:
`&.` có tác dụng gì em ơi

Reply:
dạ toán tử &. trong trường hợp này có tác dụng kiểm tra sự tồn tại của user. Nếu tồn tại thì login user và redirect to user, nếu không tồn tại thì sẽ thông báo fail

Reply:
sao lại là redirect to user em nhỉ ???

Reply:
dạ cái này để điều hướng đến url của trang thông tin user ạ